### PR TITLE
Setup Server

### PR DESCRIPTION
**Pull request by this-Cunningham**

### CODE REVIEW

**Comment by this-Cunningham** [comment]
@pachown wanna review my code ;-) (an eye for an eye?)
@sabateklu 
@collin-newman

File: trip-advisor-service/server/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+app.use(express.static('../public'));
+
+app.listen(port, (err) => {
+  if (err) {
+    console.log("Error Starting server")
+  } else {
+    console.log("Server Running on Port: ", port)
+  }
+});
```
Comment (collin-newman) [inline trip-advisor-service/server/index.js]: Airbnb eslint error. Need to add a new line at the end of each file

File: trip-advisor-service/server/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const app = express();
```
Comment (collin-newman) [inline trip-advisor-service/server/index.js:3]: Should be a line between the last 'require' and any other code per Airbnb linting

**Review by collin-newman — CHANGES_REQUESTED** [review]
Fix styling errors then it will be okay for approval.

File: trip-advisor-service/server/index.js (hunk 2 above)
Comment (this-Cunningham) [reply trip-advisor-service/server/index.js:3]: I added the changes, thanks

**Comment by this-Cunningham** [comment]
[PR SERVER](https://trello.com/c/mESqeHsD/37-pr-server)

**Review by pachown — APPROVED** [review]
Looks like a server to me. Did eslinter complain about your console.logs like it did mine?

**Comment by this-Cunningham** [comment]
[PR Server initial](https://trello.com/c/5fnooUYL/40-pr-server-initial)

**Review by collin-newman — APPROVED** [review]

**Pull request merged**